addhost

```diff
@@ -46,7 +51,7 @@ fi
 
 if [ "$cmd_do_bootstrap" = "yes" ]; then
    scp apt/cosmos_1.5-1_all.deb apt/bootstrap-cosmos.sh root@$cmd_hostname:
-   ssh root@$cmd_hostname ./bootstrap-cosmos.sh $cmd_fqdn $rrepo $rtag
-   ssh root@$cmd_hostname cosmos update 
-   ssh root@$cmd_hostname cosmos apply
+   ssh -t root@$cmd_hostname ./bootstrap-cosmos.sh $cmd_fqdn $rrepo $rtag
```

Comment:
As discussed IRL, I think the right solution is to ensure cosmos does not require a pty/tty, not to give it one during bootstrap. Otherwise, I suspect cosmos will fail later on when it runs from cron and do not have a pty/tty.

Reply:
ok but I'll merge this for now until we have a better solution